allensdk/brain_observatory/behavior/behavior_project_lims_api.py

```diff
@@ -117,13 +117,31 @@ def _build_experiment_from_session_query() -> str:
         query = f"""
             -- -- begin getting all ophys_experiment_ids -- --
             SELECT
-                (ARRAY_AGG(DISTINCT(oe.id))) as experiment_ids, os.id
+                (ARRAY_AGG(DISTINCT(oe.id))) AS experiment_ids, os.id
             FROM ophys_sessions os
             RIGHT JOIN ophys_experiments oe ON oe.ophys_session_id = os.id
             GROUP BY os.id
             -- -- end getting all ophys_experiment_ids -- --
         """
         return query
+    
+    @staticmethod
+    def _build_line_from_donor_query(line="driver") -> str:
```

Comment:
Low priority: this should probably error out if you pass a line other than "driver" or "reporter".

3.8 has such nice things: https://docs.python.org/3/library/typing.html#typing.Literal

Reply:
Normally yes, but I think here the risk is low since this is a private method. The args never actually get passed from a public method.